exonum/src/encoding/fields.rs

```diff
@@ -120,10 +125,20 @@ macro_rules! implement_std_typedef_field {
     )
 }
 
-/// Implement field helper for all POD types
-/// it writes POD type as byte array in place.
+/// Implements a field helper for all POD types. This macro enables to convert
+/// POD type data into byte array.
+///
+/// Additionally, this macro implements the
+/// [`ExonumJson`] and [`Field`] traits for data of POD type, so that they can
+/// be used as an internal entity in Exonum.
+///
+/// For additional information, refer to the [`encoding`] section.
```

Comment:
Nitpick: I'd use "module documentation" instead of "section" here and in other similar instances; I find the use of "section" slightly confusing in this context.